<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>catalog</class>
 <widget class="QDialog" name="catalog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Каталог</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Фрукты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Овощи</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Сухофрукты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Мясо и птица</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Молочные продукты</string>
   </property>
  </widget>
  <widget class="QLabel" name="ches">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Загрузки/сыр.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>220</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Сыр и колбаса</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>460</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Крупы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>710</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Хлеб и выпечка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>710</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Напитки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>460</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Сладости</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/png.png</normaloff>:/resur/mg/png.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/овощи.png</normaloff>:/resur/mg/овощи.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/сухо.png</normaloff>:/resur/mg/сухо.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/сыр.png</normaloff>:/resur/mg/сыр.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/мясо.png</normaloff>:/resur/mg/мясо.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/молок.png</normaloff>:/resur/mg/молок.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/крупы.png</normaloff>:/resur/mg/крупы.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/шок.png</normaloff>:/resur/mg/шок.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/хлеб.png</normaloff>:/resur/mg/хлеб.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resurs.qrc">
     <normaloff>:/resur/mg/напит.png</normaloff>:/resur/mg/напит.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resurs.qrc"/>
 </resources>
 <connections/>
</ui>
